server/models/Conversation.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const conversationSchema = new Schema({
+  users: [{ type: Schema.Types.ObjectId, ref: "User" }],
+  dateCreated: { type: Date, default: new Date() },
+  dateEdited: { type: Date },
+  messages: [{ type: Schema.Types.ObjectId, ref: "Message" }],
```

Comment:
This object may get really really big if a conversation had a lot of messages. I don't think we need the de-normalization of messages here, and can probably remove this array, and instead fetch messages by conversation Id that is stored in the messageSchema

Reply:
Okay the ref array has been dropped!